addons/skin.estuary/xml/DialogMediaSource.xml

```diff
@@ -63,6 +63,14 @@
 					<visible>Control.HasFocus(10)</visible>
 					<texture colordiffuse="button_focus">lists/focus.png</texture>
 				</control>
+				<control type="image">
+					<left>0</left>
+					<top>0</top>
+					<bottom>0</bottom>
+					<right>0</right>
+					<visible>!Control.HasFocus(10)</visible>
+					<texture colordiffuse="button_alt_focus">lists/focus.png</texture>
```

Comment:
Possibly a fadediffuse animation would be more efficient to change colordiffuse than using two controls

i.e.

```xml
<control type="image">
	<left>0</left>
	<top>0</top>
	<bottom>0</bottom>
	<right>0</right>
	<animation effect="fadediffuse" condition="!Control.HasFocus(10)" start="button_focus" end="button_alt_focus" reversible="false">Conditional</animation>
	<animation effect="fadediffuse" condition="Control.HasFocus(10)" start="button_alt_focus" end="button_focus" reversible="false">Conditional</animation>
	<texture>lists/focus.png</texture>
</control>
```


I'd put it in an include so that it can be reused across all the different spots i.e.

```xml

[Includes.xml]
<include name="MediaListFocusImage">
    <param name="fade_condition">false</param>
    <definition>
        <control type="image">
            <nested />
            <animation effect="fadediffuse" condition="$PARAM[fade_condition]" start="button_focus" end="button_alt_focus" reversible="false">Conditional</animation>
            <animation effect="fadediffuse" condition="![$PARAM[fade_condition]]" start="button_alt_focus" end="button_focus" reversible="false">Conditional</animation>
            <texture>lists/focus.png</texture>
        </control>
    </definition>
<include>

[focusedlayout]
<include content="MediaListFocusImage">
    <param name="fade_condition">!Control.HasFocus(10)</param>
    <left>0</left>
    <top>0</top>
    <bottom>0</bottom>
    <right>0</right>
</include>

```